client/asset/btc/btc.go

```diff
@@ -4951,11 +5031,10 @@ var dummyP2PKHScript = []byte{0x76, 0xa9, 0x14, 0xe4, 0x28, 0x61, 0xa,
 
 // EstimateSendTxFee returns a tx fee estimate for sending or withdrawing the
 // provided amount using the provided feeRate.
-func (btc *intermediaryWallet) EstimateSendTxFee(address string, sendAmount, feeRate uint64, subtract bool) (fee uint64, isValidAddress bool, err error) {
+func (btc *baseWallet) EstimateSendTxFee(address string, sendAmount, feeRate uint64, subtract bool) (fee uint64, isValidAddress bool, err error) {
```

Comment:
This makes the electrum wallets `TxFeeEstimator`s. Anything wrong with that @chappjc?

Reply:
Not in principle.  With your move of `estimateSendTxFee` to be a `baseWallet` method, it looks like the existing `(*electrumWallet).listUnspent` is adequate.